addons/website_slides/models/slide_slide.py

```diff
@@ -389,6 +389,8 @@ def _get_can_publish_error_message(self):
 
     @api.model
     def create(self, values):
+        if not 'channel_id' in values:
+            raise UserError(_("Please save the new content and the course in order to access the website with this button."))
```

Comment:
add it to the `.pot` file too

Reply:
Thanks @Yenthe666  for the review !
Could you check the update ?

Reply:
Looks good to me! 
Disclosure: I don't work at Odoo SA itself, just to be clear